dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-databend/src/test/java/org/apache/dolphinscheduler/plugin/datasource/databend/param/DatabendDataSourceProcessorTest.java

```diff
@@ -43,7 +43,7 @@ public class DatabendDataSourceProcessorTest {
     @Test
     public void testCheckDatasourceParam() {
         DatabendDataSourceParamDTO databendDataSourceParamDTO = new DatabendDataSourceParamDTO();
-        databendDataSourceParamDTO.setHost("localhost");
```

Comment:
Why change previous `localhost` to `0.0.0.0`? BTW, it would be better if you add UT cases for both ipv4 and ipv6.

Reply:
if localhost， isInetAddress  will return false and throw execption. 
At first， I want to user if(isInetAddress){}else if(source)，